run-upgrade-tests.sh

```diff
@@ -61,6 +71,8 @@ pifpaf_stop(){
 cleanup(){
     pifpaf_stop
     rm -rf $GNOCCHI_DATA
+    indexer_stop
+    [ -n $STORAGE_DAEMON ] && storage_stop
```

Comment:
https://github.com/gnocchixyz/gnocchi/blob/master/setup.cfg#L94 seems to be always enabling ceph so STORAGE_DAEMON gets set. you may want to add a clause to ensure it's not file or to remove ceph from setup.cfg (although not sure what other things gets affected)

Reply:
Ah, yeah. Thanks I'll try to do a check instead.